pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/parsers/MetsXmlParser.scala

```diff
@@ -21,6 +22,7 @@ object MetsXmlParser {
       Mets(
         recordIdentifier = id,
```

Comment:
It’s not entirely in scope for this PR, but could you go back and add these nice comments to `recordIdentifier` and `accessCondition` soon please? :)